### PR TITLE
Modify genYAML.py to create Yaml file for the snow DA

### PR DESCRIPTION
**Pull request by jiaruidong2017**
**Modified files:**
- parm/land/letkfoi/letkfoi.yaml
- test/CMakeLists.txt
- ush/ufsda/genYAML.py

**Added file:**
- test/genYAML_prep_land.sh

**Two ctests were added to the GDASApp for creating Yaml file for snow DA:**
- test_gdasapp_genYAML_land_prep
- test_gdasapp_genYAML_land_run

See issue #358

### CODE REVIEW

File: ush/ufsda/genYAML.py [hunk 1]
```diff
@@ -74,15 +74,22 @@ def get_runtime_config(config_dict):
...
         'chem': 'AERO',
         'ocean': 'SOCA',
-        'land': 'land',
+        'land': 'LAND',
     }
     win_begin_var = component_dict[config_dict.get('COMPONENT', 'atmos')] + '_WINDOW_BEGIN'
     win_end_var = component_dict[config_dict.get('COMPONENT', 'atmos')] + '_WINDOW_END'
     win_len_var = component_dict[config_dict.get('COMPONENT', 'atmos')] + '_WINDOW_LENGTH'
-    atm_begin_var = 'ATM_BEGIN_YYYYmmddHHMMSS'
+    com_begin_var = 'ATM_BEGIN_YYYYmmddHHMMSS'
```
Comment (CoryMartin-NOAA) [inline ush/ufsda/genYAML.py]: com as in common? com is an ops name/path, suggest making it `fv3` instead

**Review by CoryMartin-NOAA — COMMENTED** [review]
one small comment

File: ush/ufsda/genYAML.py (hunk 1 above)
Comment (jiaruidong2017) [reply ush/ufsda/genYAML.py]: Made changes from `com_` to `fv3_`. Thanks @CoryMartin-NOAA

**Review by CoryMartin-NOAA — APPROVED** [review]

**Review by guillaumevernieres — APPROVED** [review]
:+1:

**Pull request merged**